### DISCUSSION TITLE
snapshot: checkpoints directory not in db 0

### DISCUSSION THREAD

**Discussion by opt-m** [vesoft-inc/nebula#5484] [Q&A]
when i create a .tsnapshot, i found db 0 dont have checkpoints directory, should i backup whole db 0 or just ignore

**Comment by pengweisong** [comment]
> when i create a .tsnapshot, i found db 0 dont have checkpoints directory, should i backup whole db 0 or just ignore

0 is a special space id, it is used to store some runtime info instead of real data, you could just ignore it.
